src/shared/Microsoft.Git.CredentialManager.UI/ViewModels/WindowViewModel.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using Avalonia.Platform;
+
+namespace Microsoft.Git.CredentialManager.UI.ViewModels
+{
+    public class WindowViewModel : ViewModel
+    {
+        private bool _extendClientArea;
+        private bool _showClientChromeOverride;
+        private bool _showDebugControls;
+        private string _title;
+
+        public event EventHandler Accepted;
+        public event EventHandler Canceled;
+
+        public WindowViewModel()
+        {
+            // Default to hiding the system chrome on macOS only for now
```

Comment:
System Chrome?

Reply:
The window chrome (window title bar) as drawn by the system (as apposed to the client/our home-drawn chrome).